extensions/amp-story/0.1/amp-story-desktop.css

```diff
@@ -356,62 +352,52 @@ amp-story-page.i-amphtml-story-page-shown {
 
 .i-amphtml-story-share .i-amphtml-story-share-list {
   padding: 0!important;
-  margin: 0 80px 0 0!important;
-}
-
-.i-amphtml-story-share .i-amphtml-story-share-list > li:first-child,
-.i-amphtml-story-share .i-amphtml-story-share-list > li:last-child,
-.i-amphtml-story-share .i-amphtml-story-share-list > li {
-  padding: 0!important;
-  width: 40px!important;
-  height: 40px!important;
+  margin: 0 72px 0 0 !important;
 }
 
 [desktop] .i-amphtml-story-share .i-amphtml-story-share-list > li {
-  width: 56px!important;
-  height: 56px!important;
-}
-
-.i-amphtml-story-share .i-amphtml-story-share-list > li svg {
-  width: 28px!important;
-  height: 28px!important;
+  width: 48px !important;
+  height: 48px !important;
+  padding: 0 !important;
 }
 
-.i-amphtml-story-share:hover:before {
-  width: calc(100% + 16px)!important;
+[desktop] .i-amphtml-story-share:hover:before {
+  width: calc(100% - 32px) !important;
```

Comment:
use `transform: scale(0)` perhaps?

Reply:
probably will need some extra work with scale thing

Reply:
Yeah, I was thinking to do that but it's too complicated given the rounded corners. Would basically have to have additional elements/pseudo-elements for the "pill" ends that translate, whilst a middle element scales. Tricky. 

Since it was done this way at the beginning and the performance hit for animating `width` is negligible in this case, I'd rather leave it as it is.